### PR TITLE
Upgrade to WKWebView

### PR DESCRIPTION
**Pull request by yujingz**

### CODE REVIEW

File: ios/RNHTMLtoPDF/RNHTMLtoPDF.m [hunk 1]
```diff
@@ -157,13 +157,10 @@ - (instancetype)init
 
 }
 
-- (void)webViewDidFinishLoad:(UIWebView *)awebView
+- (void)webView:(WKWebView *)webView didFinishNavigation:(WKNavigation *)navigation
 {
-    if (awebView.isLoading)
-        return;
-
```
Comment (jiwansvmx) [inline ios/RNHTMLtoPDF/RNHTMLtoPDF.m]: ☝️ may be keep this check ?

Comment (yujingz) [reply ios/RNHTMLtoPDF/RNHTMLtoPDF.m]: 👍

**Comment by jiwansvmx** [comment]
@yujingz let's just keep the changes in this branch and change app's package.json to this this branch.

**Comment by yujingz** [comment]
@jiwansvmx we dont have to include your base url changes?

**Comment by jiwansvmx** [comment]
@yujingz looks like you branched off from `add-base-url` so it should include the changes

**Pull request closed without merging**